### DISCUSSION TITLE
ExperimentalMetadata files have timestamps in seconds, can this be in datetimes?

### DISCUSSION THREAD

**Discussion by jkbhagatio** [SainsburyWellcomeCentre/aeon_experiments#261] [General]

**Comment by glopesdev** [comment]
@jkbhagatio All data from Aeon experiments uses the Harp timestamp format which uses the internal device timestamp, which is represented in seconds from zero. Regardless of data format (e.g. CSV, harp binary, etc) this is our primary key for all tables.

When loading data with the CSV low-level API this representation is converted back to pd.Timestamp assuming the first column is following the Harp timestamp format, therefore it would be a significant breaking change to add exceptions to just a specific stream. Can you elaborate a bit further so we can understand the reasoning behind this?

**Comment by jkbhagatio** [comment]
Oh oops, I realized in one particular case I was reading the csv directly rather than using the Reader, sorry was being loony, I'll close this
